### DISCUSSION TITLE
after new OS install I get these "504 Gateway Time-out" alot

### DISCUSSION THREAD

**Discussion by rezad1393** [FreshRSS/FreshRSS#6676] [Q&A]
I lost my OS and had to install archlinuxarm from scratch again on a SD-card
.
thankfully I use nginx with its own config on a folder in  hdd and use sqlite for freshrss so I just had to install needed php stuff and then from memory and trial/error make it work.

but I get a lot of this timeouts.
first I thought maybe btrfs is at fault but my last install was on the same hdd storage.

in nginx error log I (sometimes not always) get these:
`2024/08/01 18:41:25 [error] 6647#6647: *65981 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.1.101, server: odroid, request: "GET /i/?a=normal&get=f_2549&rid=66ab8fffa0b70&state=3 HTTP/2.0", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock", host: "odroid:1081"`

**Comment by Alkarex** [comment]
You could try our Docker image https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md
Otherwise it requires some local debugging, likely independent of FreshRSS.

**Reply by rezad1393** [reply]
is that debugging to hard?
because  I think it is php (or php-fm) replated and I would like to fix it if I can (I am a noob so I may need help with that)


about the docker thing .....I know.
I tried the docker one once but I didn't know how to move the data folder of docker from SD-card to HDD.
but I to be honest quite liked the simplicity of it .
but I dont know enough docker to know if after update it can use the same data from last docker image.
